source/renderer/app/api/utils/apiHelpers.ts

```diff
@@ -27,22 +27,23 @@ export const doesWalletRequireAdaToRemainToSupportTokens = (
   hasAssetsRemainingAfterTransaction?: boolean
 ): {
   requiresAdaToRemainToSupportNativeTokens: boolean;
-  adaToRemain?: number;
+  adaToProceed?: number;
 } => {
   const adaToProceedRegex = new RegExp(
-    /.*I need approximately([\s\d.,]+)ada to proceed.*/
+    /I need approximately\s+([\d.,]+)\s+ada to proceed/
```

Comment:
Two small things:
1. The `new RegExp` construction is redundant here as you are passing a regex literal which itself is sufficient.
2. I am wondering if the number formating setting, which we can set in the profile settings applies to the number in that error message.. I guess that setting is only for our ui part and probably it is not sent to the cardano-wallet. Nevertheless, I think it is worth ensuring.